### ISSUE TITLE
Unable to record tracks

### ISSUE THREAD

**Issue by nickw7coc** [osmandapp/OsmAnd#10512]
For some time I've been unable to record tracks with the app Ver 3.8.4 on my Huawei Mate P20 Pro, Android 10.1.0.329.

I've got the default profile as Walking.

The record button is showing on the screen and when I touch it, it indicates that it's recording, though there's no recorded track shown, though when I press the button again it prompts me 4 options, and if I Save it I'm not offered any options, and therefore am not able to go off and find the file...?

Any help gratefully received :)

**Comment by jrowens** [comment]
Sorry to assume your ignorance, but, have you looked in the My Places -> Tracks screen? That's where it's saved, even if you don't click that 'Save' button (that did throw me off for a while after that update), in either the 'rec' or monthly subfolders according to the plugin setting. That's also where, if things are working correctly, you can set the 'Currently recording track' to be visible.

If you already know all about these things, then it's a trickier thing, I suppose, and beyond me.

**Comment by nickw7coc** [comment]
As you can see, there's nothing recorded, even though I have tried many times, including my dog walk yesterday... 
![Screenshot_20210112_070335_net osmand](https://user-images.githubusercontent.com/1060345/104280761-54dbe900-54a4-11eb-9031-3d5789e9680e.jpg)

**Comment by MarshCastle** [comment]
I get the same. did you find out what was going on here?

**Comment by luigifab** [comment]
Same for me: OsmAnd 4.4.7, Android **Go** 11.

The track recording is on, my position is displayed on the map (a downloaded standard map vector), I'm walking, my position is right, but no track is recorded or visible.

**Comment by sonora** [comment]
Android Go edition is a stripped-down version of the OS with among others several system services removed. I guess we would have to investigate if anything used for track recording is not present in Android Go. The capability to run a foreground service comes to mind (does navigation work? It also acts as a foreground service.) Does GPS work, i.e. do you see satellite reception, e.g. via OsmAnd's GPS widget?

**Comment by sonora** [comment]
No difference if you leave the screen on with the app in the foreground, so we can exclude a power management issue, right? And I assume your recording interval is not exceptionally long, so within 100m of walking there should be a few measurement points?

Can you try to use the 'Plan a trip' functionality to create a short track and save it, to make sure OsmAnd has no problrem writing files to the _tracks_ folder? (You should see that track then in MyPlaces=Tracks.)

**Comment by luigifab** [comment]
I tried again, screen always on, app always in the foreground, GPS reception ok (~20/49).
2 km with "Android API", 2 km wih Google Play Services.

I started the "Trip recoding" twice, and finished it twice.
During the "Trip recording", I checked my position several times, and always good.
The "Map orientation" is: "Movement direction", worked all the time.

"My Places > Tracks" still empty.
Not sure how to test if OsmAnd can write or not in the folder, but I tried to "Plan a route", and the GPX is saved in _Android/obb/net.osmand/tracks_. The GPX is present in "My Places > Tracks".

![2](https://user-images.githubusercontent.com/31816829/233772145-7385b74d-a530-42e1-8530-ca88c9b85be7.png)

**Comment by luigifab** [comment]
And yes, configuration of trip recording is:*
- general logging interval: 2s
- minimum displacement: 2m
- minimum accuracy: 1m
- minimum speed: 2 km/h

I see a button reset plugin settings, I'm trying.

**Comment by pebogufi** [comment]
Try to increase minimum accuracy to a higher value of about 50 m for test. Does it change recording ?

**Comment by luigifab** [comment]
Yes, true. I selected "Not selected" for "Minimum accuracy", and now the "Trip recording" works.
That's lol because when I set up OsmAnd a few weeks ago, I looked to display the accuracy in the top right of the map, but I couldn't find the setting.

I just tested with 50m, also working.

**Comment by pebogufi** [comment]
I made also a test with minimum accuracy =5 m. 
(My normal setting is 50 m)
For 600 m NO points recorded, OSM tracker recorded 57 points.
@sonora 
I think there should be a warning if minimum accuracy is set to less then 15 or 10 m, that this might result in very few points or no points at all recorded.

**Comment by sonora** [comment]
Yes  in fact,  filtering tracks during recording is rather critical as it (and particularly its extent) usually goes unnoticed.

I had once proposed to indicate the number of filtered points during recording, so that in this case the notification would have displayed the number of points as something like "**0 (47 filtered out)**" or similar.

It has never been implemented, but I think it still is a good idea (perhaps @vshcherb).
